### PR TITLE
Update the scripts for WSL

### PR DESCRIPTION
**Pull request by dzeyelid**
I update the scripts to be able to run on WSL.

note: I'm sorry my commit comments are not good in English. Please squash this pull request when you merge.

### CODE REVIEW

**Comment by ppremk** [comment]
@dzeyelid could you please help verify the changes made was tested and creates the necessary repos and automation please 🙇🏽

**Comment by dzeyelid** [comment]
@ppremk Thank you for helping me to improve my PR. I updated my code to resolve the `SC1091` linter suggestion.

Now, I'm considering how to respond to [the following linter suggestion](https://github.com/githubtraining/training-manual/runs/3348529607?check_suite_focus=true#step:4:96). The part is not updated by me. So I don't know whether I should respond to the linter suggestion or not.

```
2021-08-17 08:38:46 [INFO]   File:[/github/workspace/script/create-files]
2021-08-17 08:38:46 [ERROR]   Found errors in [shellcheck] linter!
2021-08-17 08:38:46 [ERROR]   Error code: 1. Command output:
------

In /github/workspace/script/create-files line 7:
source "$HOME/.trainingmanualrc"
       ^-----------------------^ SC1091: Not following: ./.trainingmanualrc: openBinaryFile: does not exist (No such file or directory)

For more information:
  https://www.shellcheck.net/wiki/SC1091 -- Not following: ./.trainingmanualr...
------
```

**Comment by ppremk** [comment]
> So I don't know whether I should respond to the linter suggestion or not.

@dzeyelid will  take a look at it. Thanks for responding 🙇🏽

**Review by Karthikjusme — APPROVED** [review]

**Review by Karthikjusme — APPROVED** [review]

**Comment by dzeyelid** [comment]
@ppremk @Karthikjusme If you can allow ignoring the linter error `SC1091` like below, we can resolve this pull request restriction. How do you think?

```diff
- # shellcheck disable=SC1090
+ # shellcheck disable=SC1090 disable=SC1091
  source "$HOME/.trainingmanualrc"
```

**Review by Ravikumartn — APPROVED** [review]

**Review by Ravikumartn — APPROVED** [review]

**Comment by dzeyelid** [comment]
@Ravikumartn Thanks for your approval!

But this pull request is still prevented by the Linter check failure.  How's my suggestion to ignore checking the line that has the problem [above](https://github.com/githubtraining/training-manual/pull/315#issuecomment-914855233)?

**Review by smarlaku820 — APPROVED** [review]
LGTM

**Review by JhoanAlvear — COMMENTED** [review]
se revian cambios, se ven aceptables

**Review by JhoanAlvear — APPROVED** [review]

**Review by JhoanAlvear — COMMENTED** [review]
DSFSDF

**Review by JhoanAlvear — COMMENTED** [review]
cambuis hechos

File: script/create-files [hunk 1]
```diff
@@ -57,7 +57,7 @@ create_files() {
       # Pick a random image for slide
       random_image=${slide_images[$RANDOM % ${#slide_images[@]}]}
       # Check the operating system to determine which commands to run
-      if [[ "$OSTYPE" = "msys" ]]; then
+      if [[ "$OSTYPE" = "msys" || "$OSTYPE" = "linux-gnu" ]]; then
```
Comment (JhoanAlvear) [inline script/create-files:60]: cambio bien hecho

**Review by JhoanAlvear — COMMENTED** [review]
cambuis hechos

File: script/expire-repo [hunk 2]
```diff
@@ -83,7 +83,7 @@ function BeNice() {
 }
 
 # Check the OS
-if [ "$OSTYPE" = "msys" ]; then
+if [ "$OSTYPE" = "msys" ] || [ "$OSTYPE" = "linux-gnu" ]; then
```
Comment (JhoanAlvear) [inline script/expire-repo:86]: revisado

File: script/new-virtual [hunk 3]
```diff
@@ -38,7 +38,7 @@ function GetDetails() {
       GITHUB_EMAIL=$( (curl -s -S -u "$TOKEN_OWNER:$TEACHER_PAT" -X GET "https://$INSTANCE_URL/users/$TOKEN_OWNER" | jq .email) | (sed -e 's/^"//' -e 's/"$//'))
       git config user.email "$GITHUB_EMAIL"
     } >>log.out 2>&1
-    if [ "$OSTYPE" = "msys" ]; then
+    if [ "$OSTYPE" = "msys" ] || [ "$OSTYPE" = "linux-gnu" ]; then
```
Comment (JhoanAlvear) [inline script/new-virtual:41]: revisado

**Review by JhoanAlvear — COMMENTED** [review]
revisado

**Review by JhoanAlvear — APPROVED** [review]

**Review by JhoanAlvear — APPROVED** [review]
se aprueban cambios

**Comment by parkerbxyz** [comment]
> @ppremk @Karthikjusme If you can allow ignoring the linter error `SC1091` like below, we can resolve this pull request restriction. How do you think?
> 
> ```diff
> - # shellcheck disable=SC1090
> + # shellcheck disable=SC1090 disable=SC1091
>   source "$HOME/.trainingmanualrc"
> ```

👋 @dzeyelid Almost! You should be able to fix the linter by doing the following:

```diff
- # shellcheck disable=SC1090
+ # shellcheck disable=SC1091
source "$HOME/.trainingmanualrc"
# shellcheck source=script/shared_functions
source ./shared_functions
```

I tried fixing this myself, but you haven’t [allowed edits from maintainers](https://docs.github.com/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests) in this PR.

**Comment by ppremk** [comment]
@dzeyelid 👋🏽  are you able to give me rights to edit your fork please? that way I can help move this along with some code fixes. Thanks 🙇🏽

**Comment by parkerbxyz** [comment]
Closing in favor of https://github.com/githubtraining/training-manual/pull/315.

**Pull request closed without merging**